### ISSUE TITLE
Why this article looks like CDDFuse

### ISSUE THREAD

**Issue by Pursuefreedomyuan** [xujian000/DAF-Net#1]
Does this paper of yours look like an improvement on CDDFuse?
The code doesn't seem to have changed much

**Comment by xujian000** [comment]
Yes, our ideas were inspired by cddFuse, and there are certain similarities with the cddFuse code. We have cited the cddFuse paper in our work. However, there are still significant differences. We incorporated domain adaptation techniques and developed innovative multi-scale kernel functions based on our task. If you're interested, you can refer to the implementation of the mkmmd section in the code for more details.

**Comment by Pursuefreedomyuan** [comment]
Thank you for your reply ! I'm sorry that I just had a cursory look at the paper and code. Although the network architecture is roughly similar to cddfuse, but the domain adaptation techniques used and developed innovative multi-scale kernel functions in the paper are very creative.In recently ,I want to improve CDDFuse ,but the video memory exceeded 24GB, which forced me to abandon the idea.I wish you have a good achievement!
